tests/unit/lms/assets_test.py

```diff
@@ -1,4 +1,5 @@
 import os
```

Comment:
This file has been extensively rewritten for a few reasons:

 * It directly patched out `pkg_resources`
 * As a result of that it didn't care if any of the filenames actually lined up, and so didn't
 
I've written this to mock far less and to actually setup and test the file reading etc.